AllReadyApp/Web-App/AllReady/Controllers/OrganizationController.cs

```diff
@@ -33,18 +33,31 @@ public IActionResult Index()
         public async Task<IActionResult> ShowOrganization(int id)
         {
             if (id <= 0)
-            { 
                 return HttpNotFound();
-            }            
 
```

Comment:
We should maintain consistent curly brace conventions throughout the application. The predominant style is to include them regardless of the length of the block so that should continue.